tools/doc/html.js

```diff
@@ -220,3 +221,16 @@ function getId(text) {
   return text;
 }
 
+function getEditLink(page) {
+  // pages that shouldn't have "Edit on Github" link
+  var ignoredPages = [
+    'all',
+    'index'
+  ];
+
+  if (ignoredPages.indexOf(page) >= 0) {
+    return '';
+  }
+
+  return '| <a href="https://github.com/nodejs/node/edit/master/doc/api/' + page + '.markdown">Edit on GitHub</a>';
```

Comment:
Sure, just pushed an update. I went with concat, because I haven't seen template strings used in that file ;)